package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.3.0-rc3",
   "description": "JavaScript library to interact with MOI Protocol via RPC API",
   "private": true,
-  "main": "packages/js-moi/dist/index.js",
-  "types": "packages/js-moi/dist/index.d.ts",
+  "main": "packages/js-moi/lib.cjs/index.js",
+  "module": "packages/js-moi/lib.esm/index.js",
   "scripts": {
+    "clean": "find . -type d \\( -name 'lib.cjs' -o -name 'lib.esm' \\) -exec rm -rf {} +",
     "build": "tsc -b && rm -rf dist",
```

Comment:
This can be helpful for authors and maintainers.

- Instead of manually removing `dist`/`lib` folders.
- Sometimes it can helpfull in removing stale file(files which are not in use, but still being tracked because of version control)